abis_mapping/utils/chunking.py

```diff
@@ -0,0 +1,39 @@
+"""Provides chunking utilities for the package"""
+
+
+# Third-Party
+import frictionless
+
+# Typing
+from typing import Optional
+
+
+def should_chunk(
```

Comment:
Probably need a test for this but I'm sure you've got that on your radar.

Reply:
@kehh Good point, missed them for some reason

Reply:
Unit tests added in 18147e7